File: GCB-Investimentos/Node.js_API/Models/tb_medicos.js
class Medico{
    constructor(Id,Nome,Email,Crm,Senha,DataNascimento,TelefoneFixo,TelefoneCelular,Cep,Logradouro,Bairro,Localidade,Uf,Numero,IdTipoUsuario){
       this.Id = Id;
       this.Nome = Nome;
       this.Email = Email;
       this.Crm = Crm;
       this.Senha = Senha;
       this.DataNascimento = DataNascimento;
       this.TelefoneFixo = TelefoneFixo;
       this.TelefoneCelular = TelefoneCelular;
       this.Cep = Cep;
       this.Logradouro = Logradouro;
       this.Bairro = Bairro;
       this.Localidade = Localidade;
       this.Uf = Uf;
       this.Numero = Numero;
       this.IdTipoUsuario = IdTipoUsuario;
    }
}

module.exports = Medico;

File: GCB-Investimentos/GCB_Care_WebApi/Models/TbMedicos.cs
using System;
using System.Collections.Generic;

namespace GCB_Care_WebApi.Models
{
    public partial class TbMedicos
    {
        public TbMedicos()
        {
            TbMedicosEspecialidades = new HashSet<TbMedicosEspecialidades>();
        }

        public int Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Crm { get; set; }
        public string Senha { get; set; }
        public DateTime? DataNascimento { get; set; }
        public string TelefoneFixo { get; set; }
        public string TelefoneCelular { get; set; }
        public string Cep { get; set; }
        public string Logradouro { get; set; }
        public string Bairro { get; set; }
        public string Localidade { get; set; }
        public string Uf { get; set; }
        public string Numero { get; set; }
        public int? IdTipoUsuario { get; set; }

        public TbTipoUsuario IdTipoUsuarioNavigation { get; set; }
        public ICollection<TbMedicosEspecialidades> TbMedicosEspecialidades { get; set; }
    }
}
